torch_uncertainty/layers/distributions.py

```diff
@@ -1,110 +1,506 @@
-from abc import ABC, abstractmethod
-
+import torch
 import torch.nn.functional as F
 from torch import Tensor, nn
-from torch.distributions import Distribution, Laplace, Normal
 
-from torch_uncertainty.utils.distributions import NormalInverseGamma
 
+def get_dist_linear_layer(dist_family: str) -> type[nn.Module]:
```

Comment:
better than a class with __new__?

Reply:
I think it's simpler than using new in a class only to return a type. But I don't mind changing it if you think it's better.

Reply:
Maybe it is, yes!